pkg/webhook/testdata/manifests.v1beta1.yaml

```diff
@@ -0,0 +1,56 @@
+
```

Comment:
just a nit: we need to `spec.preserveUnknownFields` defaults to true for v1beta in order to allow it works 1.11+ and 1.16+ as well. See: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#customresourcedefinitionspec-v1beta1-apiextensions-k8s-io